app/controllers/debriefs_controller.rb

```diff
@@ -12,7 +12,7 @@ def new
 
     if Debrief.find_by(iteration: @iteration)
       flash[:alert] = "You've already debriefed this iteration"
-      redirect_to project_iteration_url(@project, @iteration)
+      redirect_to project_iteration_url(@project, @iteration) and return
```

Comment:
The code style linter might suggest something like this, but either works.

```suggestion
      return redirect_to project_iteration_url(@project, @iteration)
```